monitors.yml

```diff
@@ -177,3 +177,24 @@ monitors:
         warning: 1.7
     query: avg(last_5m):avg:system.load.1{*} by {host} > 2
     type: metric alert
+
+  - message: |
+      {{#is_warning}}
+      less than 400M of memory on {{host.name}}
+      {{/is_warning}}
+      {{#is_alert}}
+      less than 100M of memory on {{host.name}}
+      {{/is_alert}}
+
+      @zxkuqyb@gmail.com
+      @slack-pirateshiprevenge
+    multi: true
+    name: OOM on {{host.name}}
+    options:
+      notify_no_data: false
```

Comment:
This seems like one where we should always get a response, no? So notifying on no data might be a good idea, as it also hints at other issues?